tests/integration/scanner/cascade-nmssly.test.js

```diff
@@ -0,0 +1,36 @@
+const { cascadingScan } = require('../helpers')
```

Comment:
Can you include the full scanner names in the file name (also in the sslyze one)?
The abbreviations are a bit hard to read.